### ISSUE TITLE
Unable to start Witsy 1.19

### ISSUE THREAD

**Issue by irosyadi** [nbonamy/witsy#8]
Both Witsy versions 1.19 and 1.18 encounter the following error when run, but Witsy version 1.17 runs without any problems. OS: Windows 10 Pro,  version 10.0.19045 Build 19045

```
[21984:1017/215002.391:FATAL:archive_win.cc(152)] Failed to find file integrity info for resources\app.asar
[21984:1017/215002.397:ERROR:crashpad_client_win.cc(868)] not connected
```

**Comment by nbonamy** [comment]
Should be fixed in 1.19.1 (just released). Keep me updated!

**Comment by irosyadi** [comment]
It is fixed. Thanks.
